contracts/core/SetTokenFactory.sol

```diff
@@ -36,6 +36,12 @@ contract SetTokenFactory
     // Address of the Core contract
     address public core;
 
+    /* ============ Constructor ============ */
+    
+    constructor() 
+        Authorizable(4 weeks)
```

Comment:
How does it calculate 4 weeks? Is it automatically 28 days?

Reply:
I dont think SetTokenFactory needs Authorizable, going to change to just be Core's address

Reply:
It might be more explicit to note do the number of seconds vs. using a higher level construct like weeks.

OK - will remove from SetTokenFactory

Reply:
@asoong I wonder if it makes sense to pass the Core address in the constructor. Do we imagine needing to change Core's address?